.github/workflows/nightly-review.yml

```
@@ -0,0 +1,588 @@
+      - name: Run PAL MCP consensus review (AWS Bedrock)
+        id: consensus_review_bedrock
+        if: |
+          steps.scope_selector.outputs.skip != 'true' &&
+          steps.check_providers.outputs.bedrock_available == 'true'
+        uses: anthropics/claude-code-action@v1
+        continue-on-error: true
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          use_bedrock: "true"
+          show_full_output: false
+          max_turns: 30
```

actionlint (1.7.11)

[error] input "max_turns" is not defined in action "anthropics/claude-code-action@v1". available inputs are "additional_permissions", "allowed_bots", "allowed_non_write_users", "anthropic_api_key", "assignee_trigger", "base_branch", "bot_id", "bot_name", "branch_name_template", "branch_prefix", "claude_args", "claude_code_oauth_token", "exclude_comments_by_actor", "github_token", "include_comments_by_actor", "include_fix_links", "label_trigger", "path_to_bun_executable", "path_to_claude_code_executable", "plugin_marketplaces", "plugins", "prompt", "settings", "show_full_output", "ssh_signing_key", "track_progress", "trigger_phrase", "use_bedrock", "use_commit_signing", "use_foundry", "use_sticky_comment", "use_vertex"

(action)